app/models/concerns/mods_arce_solr_document.rb

```diff
@@ -81,6 +81,13 @@ def to_mods_arce
             end
           end
         end
+        if self[:related_record_ssm]
+          xml['mods'].relatedItem(otherType: "relatedRecord") do
+            xml['mods'].titleInfo do
```

Comment:
this is what the structure needs to be, we do not need to nest identifier in title
`if self[:related_record_ssm]
          xml['mods'].relatedItem(otherType: "relatedRecord") do
              self[:related_record_ssm]&.each { |related_record| xml['mods'].identifier related_record.to_s }
          end
end`